test/health_check_test_case.py

```diff
@@ -115,7 +115,7 @@ def test_cached_health(self):
         # A successful response is obtained when all the systems are functional
         self._create_mock_queues()
         endpoint_states = self._make_endpoint_states(self.endpoints)
-        app = load_app_module(self.lambda_name())
+        app = load_app_module('service')
```

Comment:
I don't think this change is related, but for some reason this test fails for me locally with both the service and the indexer. Do you have any idea why?

Reply:
`AZUL_DEBUG=1`